### ISSUE TITLE
About results

### ISSUE THREAD

**Issue by 82018303kazuma** [Tramac/Awesome-semantic-segmentation-pytorch#2]
Please tell us the hyperparameters of the result.
It is the number of epoch.

**Comment by Tramac** [comment]
The result in ```README.md``` used ```epochs=60```

**Comment by 82018303kazuma** [comment]
The result of README can not be reproduced.
The loss does not go down even after 2000 epoch.

**Comment by 82018303kazuma** [comment]
The result of 2080 epoch



![seg_0](https://user-images.githubusercontent.com/48710645/55120405-44287a80-5139-11e9-8d12-7aded478b93f.png)

**Comment by Tramac** [comment]
Hi, I don't know if your code is the latest version. Note that the learning rate decay method is newly added in the code. Please see code ```utils.lr_scheduler.py```, and the learning rate is updated by ```cur_lr = self.lr_scheduler(cur_iters)``` in ```train.py```.
The base lr can be set 1e-2, and It is not necessary to set the epochs too big.

**Comment by 82018303kazuma** [comment]
Thank you for your reply.
I will try variously.
